pkg/vulnerabilityreport/scanner_test.go

```diff
@@ -1 +1,93 @@
-package vulnerabilityreport_test
+package vulnerabilityreport
```

Comment:
In other places we use a different package for test than SUT (with _test suffix) to make sure that we're only testing exported behavior.

Reply:
So that will rule out testing private methods, which is unfortunate in my opinion.